src/main/kotlin/edu/itmo/ilang/ir/IrProcessor.kt

```diff
@@ -33,6 +33,7 @@ interface IrProcessor<T> {
             is BoolLiteral -> processBoolLiteral(entry)
             is IntegralLiteral -> processIntegralLiteral(entry)
             is RealLiteral -> processRealLiteral(entry)
+            UninitializedLiteral -> processUninitializedLiteral(entry as UninitializedLiteral)
```

Comment:
Тут тоже можно использовать is и каст станет ненужным:

```suggestion
            is UninitializedLiteral -> processUninitializedLiteral(entry)
```

Reply:
и правда, почему то сразу так не получилось, поправил